mlflow/transformers/inference_utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+
+_INFERENCE_TASK_COMPLETIONS = "llm/v1/completions"
+_INFERENCE_TASK_CHAT = "llm/v1/chat"
+
+
+def _get_output_and_usage_from_tensor(
```

Comment:
nit: In Python conventionally, a function prefixed with an underscore `_` means a *private* function that is only used inside the module. This one is called from the other module so we shouldn't mark it with an underscore. Nothing prevents ppl to call *private* function outside the module so it's more like annotation. Also we know many files in mlflow violates this.. but it's good to keep new things clean :\